lib/game.js

```diff
@@ -1,10 +1,15 @@
+/* eslint-env node */
+
+const uuidV4 = require('uuid/v4');
+
 const { GameEngine } = require('./engine');
 
 
 class Game {
   constructor(server) {
+    this.id = uuidV4();
     this.server = server;
-    this.players = [];
+    this.players = {};
```

Comment:
I don't think we need to serialize `Game`s so this could be a `Map` for clarity.